README.md

```diff
@@ -73,6 +85,7 @@ https://www.sitepoint.com/an-interview-with-elixir-creator-jose-valim/
 + What was "_wrong_" with just writing directly in Erlang? read:
 http://www.unlimitednovelty.com/2011/07/trouble-with-erlang-or-erlang-is-ghetto.html
 
+***
```

Comment:
I think these tend to look a bit weird with the double lines caused by the titles, but not a blocker at all. We don't have these anywhere else so I might suggest we remove them for consistency.

<img width="711" alt="screen shot 2017-02-16 at 18 51 46" src="https://cloud.githubusercontent.com/assets/4185328/23036001/20de1eb4-f479-11e6-8430-9198413ec490.png">